### DISCUSSION TITLE
Changing Marker values

### DISCUSSION THREAD

**Discussion by binshadpix** [patrykandpatrick/vico#968] [Questions]
### Question

is there any way to customise marker values?

### Vico version(s)

1.12.0

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @binshadpix. Please be more specific. What kind of `Marker`? A custom implementation, or `MarkerComponent`? What exactly do you mean by "marker values"? Do you mean the label?
